pennylane/devices/default_mixed.py

```diff
@@ -0,0 +1,152 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+The default.mixed device is PennyLane's standard qubit simulator for mixed-state computations.
+
+It implements the necessary :class:`~pennylane._device.Device` methods as well as some built-in
+:mod:`qubit operations <pennylane.ops.qubit>`, providing a simple mixed-state simulation of
+qubit-based quantum circuits.
+"""
+
+# tolerance for numerical errors
+tolerance = 1e-10
+
+
+class DefaultMixed(QubitDevice):
```

Comment:
Anyone know how to avoid these `undefined-variable` errors from being raised? :thinking:

Reply:
@ixfoduap it looks like QubitDevice is not being imported in this file?

Reply:
Thanks Josh! Fixed now :blush: